.jenkins/win-build.sh

```diff
@@ -31,8 +31,9 @@ cat >ci_scripts/build_pytorch.bat <<EOL
 set PATH=C:\\Program Files\\CMake\\bin;C:\\Program Files\\7-Zip;C:\\curl-7.57.0-win64-mingw\\bin;C:\\Program Files\\Git\\cmd;C:\\Program Files\\Amazon\\AWSCLI;%PATH%
 
 :: Install MKL
-aws s3 cp s3://ossci-windows/mkl.7z mkl.7z --quiet && 7z x -aoa mkl.7z -omkl
-set LIB=%cd%\\mkl;%LIB%
+aws s3 cp s3://ossci-windows/mkl_with_headers.7z mkl.7z --quiet && 7z x -aoa mkl.7z -omkl
```

Comment:
This file has a different structure than `mkl.7z`, but a worse name. If this merges, I'll ask @yf225 to rename the file on s3, and submit a PR to change it back to `mkl.7z` here.